api/src/test/java/org/apache/iceberg/util/RandomUtil.java

```diff
@@ -225,4 +229,8 @@ private static BigInteger randomUnscaled(int precision, Random random) {
 
     return new BigInteger(sb.toString());
   }
+
+  private static JsonVariant randomVariant() {
+    return JsonVariant.of("{\"name\": \"John\"}");
```

Comment:
This is not very random ;)

Reply:
Yeah. :) Let me remove this part for now. It turns out it's to be used by other module and I will add random Variant when it's getting used.